Libraries/LLM/Evaluate.swift

```diff
@@ -195,20 +285,17 @@ public func generate(
     {
         // compute the timing for the prompt
         if tokens.isEmpty {
-            eval(token)
```

Comment:
Switched to return Int -- returning the MLXArray is sketchy at best since it had not been evaluated.  Have to be _very_ careful with thread.  Anyway, this is internal to the `generate()` function.